docs/index.md

```diff
@@ -14,7 +14,14 @@ Unity Catalog is the industry’s only universal catalog for data and AI.
 
 - **Unified governance** for data and AI - Govern and secure tabular data, unstructured assets, and AI assets with a single interface.
 
-The first release of Unity Catalog focuses on a core set of APIs for tables, unstructured data, and AI assets.  More will come soon on governance, access, and client interoperability. This is just the beginning!
+The first release of Unity Catalog focused on a core set of APIs for tables, unstructured data, and AI assets. The [second release (v0.2)](https://github.com/unitycatalog/unitycatalog/releases/tag/v0.2.0) has added:
```

Comment:
Could you reword this so we no longer refer to UC releases since it's the docs?  Just call out the overall main components as of the latest release?